boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts

```diff
@@ -408,6 +408,13 @@ arduino_serial: &flexcomm12 {
 	 * memory.
 	 */
 	dma-channels = <37>;
+	nxp,dma-num-of-otrigs = <4>;
```

Comment:
the reason is because otherwise the channel chaining will only work on the EVK unless the custom board maker will add the same line, which is just the point of a dtsi for the SOC

Reply:
I have moved this to the soc level